Update apt cache if cache_valid_time is set

willthames opened the pull request:
##### SUMMARY
It's not clear from the docs whether you need to set `update_cache`
when using `cache_valid_time`.

Setting `cache_valid_time` should imply `update_cache`. Update docs
to reflect this.

##### ISSUE TYPE
 - Bugfix Pull Request

##### COMPONENT NAME
apt

##### ANSIBLE VERSION
```
ansible 2.4.0 (devel 24e393aef1) last updated 2017/08/02 14:02:31 (GMT +1000)
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/home/will/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /home/will/src/ansible/lib/ansible
  executable location = /home/will/src/ansible/bin/ansible
  python version = 2.7.13 (default, May 10 2017, 20:04:28) [GCC 6.3.1 20161221 (Red Hat 6.3.1-1)]
```

gundalow commented on lib/ansible/modules/packaging/os/apt.py:
```diff
@@ -48,7 +48,7 @@
   cache_valid_time:
     description:
-      - Update the apt cache if its older than the I(cache_valid_time). This option is set in seconds.
+      - Update the apt cache if its older than the I(cache_valid_time). This option is set in seconds. Implicitly sets I(update_cache) if set.
```
Worth adding a note saying "from Ansible 2.4..."

resmo commented:
LGTM (with version added as mentioned by @gundalow )

Commit:
Update cache if cache_valid_time is set

It's not clear from the docs whether you need to set `update_cache`
when using `cache_valid_time`.

Setting `cache_valid_time` should imply `update_cache`. Update docs
to reflect this.

willthames commented:
Thanks @gundalow. Amended to add that it's from Ansible 2.4
